lib/Runtime/Base/FunctionBody.h

```diff
@@ -2428,6 +2431,9 @@ namespace Js
         FieldWithBarrier(bool) m_hasBailoutInstrInJittedCode : 1; // Indicates whether function has bailout instructions. Valid only if hasDoneCodeGen is true
         FieldWithBarrier(bool) m_pendingLoopHeaderRelease : 1; // Indicates whether loop headers need to be released
         FieldWithBarrier(bool) hasExecutionDynamicProfileInfo : 1;
+#ifdef _M_X64_OR_ARM64
+        FieldWithBarrier(bool) m_tag : 1;                     // Used to tag the low bit to prevent possible GC false references
+#endif
```

Comment:
Why moving "m_tag"... is it because x64 difference with x86? If so would this make x86 worse?

Also this should be a build break on x86, as now "m_tag" only declared on x64 but initialized on both. (I'm reading in commit order, this could be fixed in later commit.)